### PR TITLE
Scrum 69 delete call by Id

### PR DESCRIPTION
**Pull request by AlisonGuzman2006**
This pull request introduces a comprehensive suite of test cases for meeting deletion functionality within the EspoCRM API. The focus is to ensure robustness and reliability of the delete operation under various scenarios. The tests are designed to validate proper API behavior, including successful deletion, handling of invalid requests, and correct response codes.

Setup Method:
Function: setup_create_meeting
Purpose: Creates a call before each test to ensure there is a valid call is available for deletion.

Details:

Generates headers using valid credentials.
Creates a meeting and yields its ID along with the headers for use in the test.
Test Cases Implemented:
Test Method: test_delete_call_successful => Expected Result: 200 OK
Test Method: test_delete_call_header_format_content_type =>expected Result: Valid content type in headers from response
Test Method: test_delete_more_than_one_call => Expected Result: 200 OK, but can´t execute because it uses a wrong verb "POST" and not "DELETE"
Test Method: test_delete_unexisting_call=>Expected Result: 404 Not Found
Test Method: test_delete_invalid_call_id => Expected Result: 400 Bad request
Test Method: test_delete_call_with_invalid_authentication => Expected Result: 401 Unauthorized
Test Method: test_delete_call_with_invalid_cookie => Expected Result: 401 Unauthorized

Test Method: test_delete_meeting_confirmation_response_body
Expected Result: Response body 'true'
Ensures: Proper response body format.

### CODE REVIEW

File: api_endpoints/api_request.py
Comment (Diego-Claros17) [inline api_endpoints/api_request.py:1]: Good but be carefull with static methods

File: assertions/assertion_calls.py
Comment (Diego-Claros17) [inline assertions/assertion_calls.py:1]: GJ

File: tests/calls/conftest.py
Comment (Diego-Claros17) [inline tests/calls/conftest.py:1]: Some problems were found but it was solved by removing the use of staticmethod and calling the methods directly.

File: tests/calls/test_delete_call.py
Comment (Diego-Claros17) [inline tests/calls/test_delete_call.py:1]: Clean code and all test run correctly

File: tests/calls/test_post_call.py
Comment (Diego-Claros17) [inline tests/calls/test_post_call.py:1]: OK

**Review by Diego-Claros17 — APPROVED** [review]
This PR does not affect the code and its well writed and run as expected. Well done Ali

**Review by ErickaIsa — DISMISSED** [review]
The code and tests look good.

**Review by Diego-Claros17 — DISMISSED** [review]
All works as expected. Well done Ali

File: api_endpoints/api_request.py
Comment (Diego-Claros17) [reply api_endpoints/api_request.py:1]: A conflict with a duplicated method but it was resolved

**Review by Diego-Claros17 — APPROVED** [review]
All works well, all conflicts were solved. GJ Ali

**Review by ErickaIsa — APPROVED** [review]
All conflicts is resolved

**Pull request merged**